sdk/cmake/host-tools.cmake

```diff
@@ -27,17 +27,23 @@ function(configure_host_tools HOST_TOOLS_DIR)
 
     set_property(SOURCE host_tools PROPERTY SYMBOLIC 1)
 
+    include(${HOST_TOOLS_DIR}/ImportExecutables.cmake)
+    include(${HOST_TOOLS_DIR}/TargetList.cmake)
+
+    foreach(_target ${NATIVE_TARGETS})
+        get_target_property(_target_location native-${_target} IMPORTED_LOCATION_DEBUG)
+        list(APPEND _target_locations ${_target_location})
```

Comment:
Does cmake cache the `_target_locations` in between runs?
e.g. does it grow each re-configure?

Reply:
No, otherwise the same would be true of `SOURCE` in each directory.